templates/react-ecs/mprocs.yaml

```
@@ -8,3 +8,16 @@ procs:
   anvil:
     cwd: packages/contracts
     shell: anvil --base-fee 0 --block-time 2
+  indexer:
+    shell: rimraf $SQLITE_FILENAME && pnpm sqlite-indexer
+    env:
+      RPC_HTTP_URL: "http://127.0.0.1:8545"
+      FOLLOW_BLOCK_TAG: "latest"
+      SQLITE_FILENAME: "indexer.db"
+  explorer:
+    cwd: /Users/karolis/Code/Lattice.xyz/experiments/mud-2/packages/explorer
```

prob wnat something like this?
```suggestion
    cwd: ../../packages/explorer
```

---

ah damn, this should not have been added now, removing now. But generally yes, I was using it for testing some stuff